pvlib/bifacial/losses_models.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+
+from typing import Literal
+
+
+def power_mismatch_deline(
+    rmad,
+    model: Literal[
+        "fixed-tilt", "single-axis-tracking"
+    ] = "single-axis-tracking",
+    fillfactor: float = None,
+):
+    r"""
+    Estimate the power loss due to irradiance non-uniformity.
```

Comment:
```suggestion
    Estimate power loss due to irradiance non-uniformity.
```
Could be nice perhaps to add "DC" as in "DC power loss"